deps/rust/BUILD.bazel

```diff
@@ -70,31 +73,44 @@ crates_vendor(
     ],
 )
 
-# TODO(cleanup): Switching to rust_library() would reduce the overhead of this by not creating a
-# crate-of-crates but this previously broke the Windows build.
+# define the path to the lolhtml headers, this is clunky but allows us to avoid cloning the
+# repository twice.
+alias(
+    name = "lol_html_api",
+    actual = "@crates_vendor__lol_html-1.2.1//:c-api/include/lol_html.h",
+    tags = ["manual"],
+    visibility = ["//visibility:public"],
+)
+
+# windows build fixes due to rust libraries missing symbols:
+# tps://github.com/bazelbuild/rules_rust/blob/144d34fd/ffi/cc/global_allocator_library/BUILD.bazel#L3
```

Comment:
Fix the url, and maybe add a comment saying that we use empty_lib to have a Rust "staticlib" target as this allows us to the missing symbols on Windows (otherwise having emptylib could be confusing)